### PR TITLE
feat: Auto-join party invites

### PR DESCRIPTION
**Pull request by magicus**
Also fixes a bug with broken party kick regex.

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/players/PartyModel.java [hunk 1]
```diff
@@ -396,6 +407,13 @@ public void partyCreate() {
         McUtils.sendCommand("party create");
     }
 
+    /**
+     * Join another players party
+     */
+    public void partyJoin(String playerName) {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/players/PartyModel.java:413]: joinParty would be a bit nicer imo

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 2]
```diff
@@ -43,6 +43,9 @@
   "feature.wynntils.autoApplyResourcePack.name": "Auto Apply Resource Pack",
   "feature.wynntils.autoApplyResourcePack.packHash.description": "Last Wynncraft Pack Hash",
   "feature.wynntils.autoApplyResourcePack.packHash.name": "Pack Hash",
+  "feature.wynntils.autoJoinParty.name": "Autojoin Parties",
```
Comment (kristofbolyai) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]:
```suggestion
  "feature.wynntils.autoJoinParty.name": "Auto Join Parties",
```

or 

```suggestion
  "feature.wynntils.autoJoinParty.name": "Auto-join Parties",
```

?

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 3]
```diff
@@ -43,6 +43,9 @@
...
+  "feature.wynntils.autoJoinParty.onlyFriends.description": "Should we only autojoin party invites from friends?",
+  "feature.wynntils.autoJoinParty.onlyFriends.name": "Only autojoin friends' parties",
```
Comment (kristofbolyai) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]: same with these too, whatever we pick above

**Review by kristofbolyai — DISMISSED** [review]
After the renames, looks good.

File: common/src/main/java/com/wynntils/models/players/PartyModel.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/models/players/PartyModel.java:413]: Maybe, but I'm following a pre-existing naming pattern here for methods.

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**